lib/run/index.js

```diff
@@ -26,6 +26,7 @@ var fs = require('fs'),
         request: ['response', 'request', 'item', 'cookies'],
         beforeTest: ['events', 'item'],
         test: ['executions', 'item'],
+        io: ['trace', 'response', 'request', 'cookies'],
```

Comment:
Arguments depend on the type of IO event.. check what "trace" contains. A request, response and cookies only make sense for HTTP IO.